app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -162,4 +162,21 @@ public function validateBeforeOrderCommit($observer) {
         $orderModel = Mage::getModel('boltpay/order');
         $orderModel->validateBeforeOrderCommit($observer);
     }
+
+    /**
+     * @param Varien_Event_Observer $observer
+     * @return Varien_Object
+     */
+    public function checkCheckoutTypeForFirecheckoutModule($observer)
```

Comment:
We don't want or need to create observers for our own self-created events.  If we find ourselves doing this, the question we should then ask ourselves is in which model or helper would this function better fit?

This pattern only needlessly obscures the control flow logic and makes it harder on code hinting via IDE's

A second observation is that if you are going to be adding event methods, in the comments, leave the name of the event or events that trigger its execution.